### PR TITLE
escape html in error detail (in WMStats)

### PR DESCRIPTION
**Pull request by ticoann**
For the display like this

```
Could not find jobReport

========== condor.26989.0.err ==========

========== condor.26989.0.out ==========

========== condor.26989.0.log ==========
Adding end of condor.26989.0.log to error message:

    <a n="SubmitHost"><s>&lt;128.142.132.152:4080?addrs=128.142.132.152-4080&amp;noUDP&amp;sock=6978_e7f4_101&gt;</s></a>
    <a n="Proc"><i>0</i></a>
    <a n="Cluster"><i>26989</i></a>
    <a n="EventTime"><s>2017-11-06T17:45:06</s></a>
    <a n="Subproc"><i>0</i></a>
    <a n="EventTypeNumber"><i>0</i></a>
</c>
<c>
    <a n="Proc"><i>0</i></a>
    <a n="TriggerEventTypeName"><s>ULOG_SUBMIT</s></a>
    <a n="EventTime"><s>2017-11-06T17:45:06</s></a>
    <a n="EnteredCurrentStatus"><i>1509986706</i></a>
    <a n="ExtDESIRED_Sites"><s>T1_US_FNAL,T2_CH_CERN</s></a>
    <a n="SubmitHost"><s>&lt;128.142.132.152:4080?addrs=128.142.132.152-4080&amp;noUDP&amp;sock=6978_e7f4_101&gt;</s></a>
    <a n="QDate"><i>1509986706</i></a>
    <a n="TriggerEventTypeNumber"><i>0</i></a>
    <a n="WMAgent_JobID"><i>1152</i></a>
    <a n="MyType"><s>JobAdInformationEvent</s></a>
    <a n="Cluster"><i>26989</i></a>
    <a n="JobStatus"><i>1</i></a>
    <a n="Subproc"><i>0</i></a>
    <a n="EventTypeNumber"><i>28</i></a>
    <a n="DESIRED_Sites"><s>T1_US_FNAL,T2_CH_CERN</s></a>
</c>
<c>
    <a n="Reason"><s>via condor_rm (by user cmst1)</s></a>
    <a n="MyType"><s>JobAbortedEvent</s></a>
    <a n="Proc"><i>0</i></a>
    <a n="Cluster"><i>26989</i></a>
    <a n="EventTime"><s>2017-11-06T17:47:46</s></a>
    <a n="Subproc"><i>0</i></a>
    <a n="EventTypeNumber"><i>9</i></a>
</c>
<c>
    <a n="Proc"><i>0</i></a>
    <a n="TriggerEventTypeName"><s>ULOG_JOB_ABORTED</s></a>
    <a n="EventTime"><s>2017-11-06T17:47:46</s></a>
    <a n="EnteredCurrentStatus"><i>1509986866</i></a>
    <a n="ExtDESIRED_Sites"><s>T1_US_FNAL,T2_CH_CERN</s></a>
    <a n="QDate"><i>1509986706</i></a>
    <a n="TriggerEventTypeNumber"><i>9</i></a>
    <a n="WMAgent_JobID"><i>1152</i></a>
    <a n="MyType"><s>JobAdInformationEvent</s></a>
    <a n="Cluster"><i>26989</i></a>
    <a n="JobStatus"><i>3</i></a>
    <a n="Reason"><s>via condor_rm (by user cmst1)</s></a>
    <a n="Subproc"><i>0</i></a>
    <a n="EventTypeNumber"><i>28</i></a>
    <a n="DESIRED_Sites"><s>T1_US_FNAL,T2_CH_CERN</s></a>
</c>
```

### CODE REVIEW

**Comment by ticoann** [comment]
@amaltaro, Alan please review

**Comment by ticoann** [comment]
test this please

**Comment by ticoann** [comment]
test this please

File: src/couchapps/WMStats/_attachments/js/WMStats.Utils.js [hunk 1]
```diff
@@ -260,3 +260,20 @@ WMStats.Utils.formatStateTransition = function(stateTransition) {
...
+    '>': '&gt;',
+    '"': '&quot;',
+    "'": '&#39;',
+    '/': '&#x2F;',
+    '`': '&#x60;',
+    '=': '&#x3D;'
+};
+
+WMStats.Utils.escapeHtml = function(string) {
+    return String(string).replace(/[&<>"'`=\/]/g, function (s) {
```
Comment (amaltaro) [inline src/couchapps/WMStats/_attachments/js/WMStats.Utils.js]: I was going to say there is a close parenthesis missing here, but it's there in the very end.
What a horrible syntax!! I'm glad you fix all these javascript problems :)

**Pull request merged**